torchrl/collectors/collectors.py

```diff
@@ -175,7 +175,7 @@ class SyncDataCollector(_DataCollector):
     Generic data collector for RL problems. Requires and environment constructor and a policy.
 
     Args:
-        create_env_fn (Callable), returns an instance of _EnvClass class.
+        create_env_fn (Callable), returns an instance of EnvStateful class.
```

Comment:
Do we want the data collector to work only with stateful envs?

If we incorporate Brax for instance, envs won't be stateful but still we may want to use them.